docs/user/openstack/README.md

```diff
@@ -199,6 +279,64 @@ Wait for the `<cluster name>-api` server comes up and you can make your changes
 Even if the installer times out, the OpenShift cluster should still come up. Once the bootstrapping process is in place, it should all run to completion.
 So you should be able to deploy OpenShift without any floating IP addresses and DNS records and create everything yourself after the cluster is up.
 
+## Installing with Kuryr SDN
+
+To deploy with Kuryr SDN instead of the default OpenShift SDN, you simply need
+to modify the `install-config.yaml` file to include `Kuryr` as the desired
+`networking.networkType` and proceed with the same steps as with the default
+OpenShift SDN:
+
+```yaml
+apiVersion: v1
+baseDomain:
+clusterID:
+controlPlane:
+  name: master
+  platform: {}
+  replicas:
+compute:
+  name: worker
```

Comment:
This needs to be a list.

Alternatively, to not have to duplicate all of the `install-config.yaml` file or have weird nil values, you could simply have:

```yaml
apiVersion: v1
...
networking:
  networkType: Kuryr
  ...
platform:
  openstack:
    ...
    trunkSupport: true
    octaviaSupport: true
    ...
```